packages/web-components/src/components/va-notification/va-notification.tsx

```diff
@@ -0,0 +1,126 @@
+import { 
+  Component, 
+  Element,
+  Event,
+  EventEmitter,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Notification
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-notification',
+  styleUrl: 'va-notification.scss',
+  shadow: true,
+})
+export class VaNotification {
+  @Element() el!: any;
+
+  /**
+   * If `true`, the card will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * Symbol indicates type of notification 
+   * Current options are: action-required, update
+   */
+  @Prop() symbol?: string = 'none';
```

Comment:
Will there ever be other icons besides action-required and update? If someone passes another icon value into this prop, would it display any Font Awesome icon that matches that class name?

Reply:
we don't anticipate at this time other types of icons, but I defer to @allisonlu as to whether it would be possible to do so if someone wanted to